0500_rust.sh

```diff
@@ -0,0 +1,19 @@
+
+# shellcheck shell=bash
+export _coreutils="@coreutils@"
+# ============================================================================ #
+#
+# Setup Rust
+#
+# ---------------------------------------------------------------------------- #
+
+# Only run if 'rustLibSrc' is in the environment
+if [[ -x "$FLOX_ENV/rustc-std-workspace-std" ]]; then
```

Comment:
Although I think it works, I don't think you want to use `-x`, do you mean `-d`?